UPGRADE.md

```diff
@@ -1,6 +1,17 @@
 eZ Platform Site API upgrade instructions
 =========================================
 
+Upgrading from 2.2 to 2.3
+-------------------------
+
+Controllers that extend from `Netgen\Bundle\EzPlatformSiteApiBundle\Controller\Controller` and are registered inside dependency injection container must set `parent` config key within service definition, example:
```

Comment:
Why is this needed?

Reply:
This would be the solution to your question about abstract trait method. A minor BC break, yes, but it would not require an additional implementation to use the trait, it would already be implemented with the trait and called when instatiating the controller. Hence requirement to use the correct parent in service definition.

Reply:
It should be explicitly worded, as far as I understand this is really about requiring `Site` injection through `setSite()` method.